spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Contestand Index', type: :feature do
+  describe 'as a user' do
+    it 'when I visit contestant index page, I should see' do
+      recycled_material_challenge = Challenge.create(theme: 'Recycled Material', project_budget: 1000)
+      news_chic = recycled_material_challenge.projects.create(name: 'News Chic', material: 'Newspaper')
+      jay = Contestant.create(name: 'Jay McCarroll', age: 40, hometown: 'LA', years_of_experience: 13)
+      ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id)
+
+      visit '/contestants'
+
+      expect(page).to have_content(jay.name)
+      expect(page).to have_content(jay.projects.first.name)
+    end
```

Comment:
I'd like to see more robust testing. Showing more than one contestant for a contestant index page is a good idea to make sure multiple names will be formatted correctly. I'd also make at least one of the contestants have multiple projects, again, so you can ensure that your view is formatting your data correctly. For example, how would your view work if jay here had 4 projects he was part of?